gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineVersionsIntegrationTest.groovy

```diff
@@ -97,9 +97,9 @@ class BaselineVersionsIntegrationTest  extends AbstractPluginTest {
         result
     }
 
-    def buildAndFailWith(String error) {
+    void buildAndFailWith(String error) {
         BuildResult result = with('checkVersionsProps', '--stacktrace').buildAndFail()
-        result.output.contains(error)
+        assert result.output.contains(error)
```

Comment:
better error messages! groovy will actually destructure your command when you use assert, just like it does when putting boolean expressions in a `then:` / `expect:` block.